feat(navbar): add the searchbar

0m3ga13 opened the pull request:
_Changes:_ 
- implemented a functional collapsible Search bar

**Screenshots**
![image](https://github.com/202306-NEA-DZ-FEW/student-store-team-2/assets/62702326/c4174b23-b8a5-43c6-a7c2-55bc34932d84)
 
fix #18

# Related Issue

- Resolve #18

Bolphunga committed:
feat: add product card

#23

Bolphunga committed:
feat(productcard): add product card and its test

#23

Bolphunga committed:
feat(productcard): add product card with its test

#23

Bolphunga committed:
Merge branch 'develop' into 23-product-card

samiraTbl committed:
feat(footer.jsx): creation the footer comonent

Footer #20

samiraTbl committed:
feat(howitworks.jsx): designing a section component

Design features section component #21

samiraTbl committed:
style(appfunctionalitydescription): i changed the name of the file component

i changed the name of the componenet from hiw to AppFunctionalityDescription

designing a section component #33

Bolphunga committed:
Merge branch 'develop' into 23-product-card

Bolphunga committed:
refactor: conflict resolve

Bolphunga committed:
feat(product card): add product card

#23

Bolphunga committed:
Merge branch 'develop' into 23-product-card

Emybel committed:
feat(button component): create and style buy and borrow  button component

re #14

0m3ga13 committed:
feat(navbar): add the searchbar

fix #18

Bolphunga committed:
refactor(productcard): fixed styling

samiraTbl reviewed (approved):
done.!

Bolphunga reviewed (approved):
Looks fantastic. Good job.

0m3ga13 committed:
feat(home page): add testimonials

in this changes we created the testimonials component as a slider, also we created starts rating

fix #32

medshk reviewed (approved):
PR permitted

0m3ga13 committed:
refactor(prettier): fix home page

Bolphunga committed:
Merge branch 'develop' into 20-footer

Bolphunga committed:
Merge pull request #38 from 202306-NEA-DZ-FEW/32-testimonial-carousel

feat(home page): add testimonials

Bolphunga committed:
Merge branch 'develop' into 23-product-card

Bolphunga committed:
Merge pull request #28 from 202306-NEA-DZ-FEW/23-product-card

23 product card

Emybel committed:
style(button component): removed unsused imports

fix #&'

Emybel committed:
chore(button component): rename file and update test

fix #14

Emybel committed:
test(button component): recreate test

fix #14

Emybel committed:
test(buton component): update test

fix #14

Bolphunga committed:
Merge branch 'develop' into 14-buyborrow-button

Bolphunga committed:
Merge pull request #35 from 202306-NEA-DZ-FEW/14-buyborrow-button

feat(button component): create and style buy and borrow button component

Bolphunga committed:
Merge branch 'develop' into 20-footer

Bolphunga committed:
feat(howitworks): added how it works section

Bolphunga committed:
Merge branch 'develop' into 21-design-features-section-component

Bolphunga committed:
Merge pull request #33 from 202306-NEA-DZ-FEW/21-design-features-section-component

feat(howitworks.jsx): designing a section component

Bolphunga committed:
Merge branch 'develop' into 20-footer

Bolphunga committed:
feat(footer): added a footer

#20

Bolphunga committed:
Merge pull request #34 from 202306-NEA-DZ-FEW/20-footer

feat(footer.jsx): create the footer component

0m3ga13 committed:
style(navbar): change the position of the navbar to absolute

0m3ga13 committed:
style(navbar): change the position of the navbar to absolute

0m3ga13 committed:
refactor(search bar): resolve conflicts

0m3ga13 committed:
refactor(searchbar): resolve conflicts
